### ISSUE TITLE
Support returning entity as a watch event result

### ISSUE THREAD

**Issue by kasisnu** [strava/go.serversets#19]
It would be useful if the raw `entity` struct could be returned. It would enable the use of `additionalEndpoints`. If there is interest, I'd be happy to take a shot at it.

**Comment by paulmach** [comment]
Yeah, it would probably be better for [Endpoints](https://github.com/strava/go.serversets/blob/master/watch.go#L121) to return []Entity.  That way you can get to that info if needed.

I'd happily review a pull request :)

**Comment by kasisnu** [comment]
Would modifying `Endpoints`' signature be a safe change?

**Comment by paulmach** [comment]
You're right, add a `EndpointEntities()` is probably a better idea.

I'm not really sure how others are using the information, but we're only interested in the list of host:ports for servers that can accept requests, each of which will register themselves. We don't do anything with the different non-Alive statuses or the additional endpoints.
